src/blockchain/tx-history-service-old.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-const { Logger } = require('common/logger');
+const { Logger } = require('logger/logger');
```

Comment:
@sk91 you know that common is shared between processes right? How will we access logger from the renderer one?